packages/lab/src/form-field-next/FormFieldHelperText.tsx

```diff
@@ -1,21 +1,19 @@
 import clsx from "clsx";
 import { makePrefixer, Text, TextProps } from "@salt-ds/core";
-import { FormFieldProps } from "./FormFieldNext";
 
 import "./FormFieldHelperText.css";
 
 const withBaseName = makePrefixer("saltFormFieldHelperText");
 
 export interface FormFieldHelperTextProps
   extends Omit<TextProps<"label">, "variant" | "styleAs"> {
-  helperText: FormFieldProps["helperText"];
   disabled?: boolean;
 }
 
 export const FormFieldHelperText = ({
```

Comment:
I think now we need to pass `disabled` down via context from the FormField, otherwise it looks like consumers will need to set it individually on FormField, HelperText and Label to keep it in sync

Reply:
true ... do we keep the prop on helper text as well, in case the component is used alone?

Reply:
I was thinking these components would only be used in FF is there another use-case?

I think we would only keep the prop if there was a case for having a different `disabled` state on the helpertext/label to the rest of the FormField, and then it could be used to override the context. But I can't imagine why that would be necessary